babel.config.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Let's use a beautifier to make this code easier to read: 

https://jsonformatter.curiousconcept.com/

Reply:
@flora8984461 @autumnfound Will we need to add this file to all our projects? Is there a way to refer to the version from the solstice-assets package: https://github.com/EclipseFdn/solstice-assets/blob/master/babel.config.json

Reply:
updated